autoshop/core/management/commands/register_profile.py

```diff
@@ -0,0 +1,20 @@
+from core.models import Profile
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Creating profiles for users that dont have a profile.'
+
+    def handle(self, *args, **options):
+        all_user_ids = User.objects.values_list('id', flat=True)
+        all_profile_ids = Profile.objects.values_list('user_id', flat=True)
+        users_with_unregistered_profiles = list(set(all_user_ids) - set(all_profile_ids))
+        if not users_with_unregistered_profiles:
+            self.stdout.write(self.style.SUCCESS('All users have profiles.'))
```

Comment:
add a method to create missing profiles of users`create_profile_for_users(self, users)` and move this logic there.